.github/workflows/reactjs_test.yaml

```diff
@@ -0,0 +1,26 @@
+name: React.js CI/CD
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - reactjs/**
+      - .github/workflows reactjs*.yaml
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+        matrix:
+            node-version: [10.x, 12.x, 14.x, 16.x, 18.x]
```

Comment:
for now, just use spesific version of node, i think its enough

Reply:
got it thanks, I will fix it.